sprout/appliances/tasks.py

```diff
@@ -907,6 +907,10 @@ def appliance_power_on(self, appliance_id):
             appliance.set_status("Powering on.")
             appliance.provider_api.start_vm(appliance.name)
             self.retry(args=(appliance_id, ), countdown=20, max_retries=40)
+
+        # fixes time synchronization
+        cfme_appliance = CFMEAppliance(appliance.provider_name, appliance.name)
```

Comment:
Look in models.py, I bet there is an IPAppliance invocation in one of the properties. Otherwise I think it is good.